tests/Unit/Models/PostModelTest.php

```diff
@@ -153,26 +153,6 @@ public function testBlinkPayload()
         );
     }
 
-    /**
-     * Test expected payload when various attributes are not set.
-     *
-     * @return void
-     */
-    public function testBlinkPayloadForNullValues()
-    {
-        $post = factory(Post::class)->create();
-
-        $result = $post->toCustomerIoPayload();
-
-        $this->assertEquals($result['group_id'], null);
-        $this->assertEquals($result['group_name'], null);
-        $this->assertEquals($result['group_type_id'], null);
-        $this->assertEquals($result['group_type_name'], null);
-        $this->assertEquals($result['school_id'], null);
-        $this->assertEquals($result['school_name'], null);
-        $this->assertEquals($result['referrer_user_id'], null);
-    }
-
```

Comment:
Since we no longer explicitly return each field with a null value, we can't `assertEquals` here. I think this test is safe to omit since the behavior & likely failure scenarios (exceptions if we omit a group) would be caught by other tests.